### PR TITLE
After password reset: Only list projects when not superadmin

### PR DESCRIPTION
**Pull request by inaod568**
- [x] Tests passing
- [x] Black formatting
- [ - ] Migrations for any changes to the database schema
- [x] Rebase/merge the `dev` branch
- [ - ] Note in the CHANGELOG

### CODE REVIEW

File: dds_web/templates/user/password_reset_completed.html [hunk 1]
```diff
@@ -15,21 +15,25 @@ <h1>Password updated!</h1>
...
+            {% endif %}
+            <small class="text-muted ml-2">
+                <a href="{{ url_for('auth_blueprint.login')
+                    }}">
+                    Back to Login
+                </a>
+            </small>
 
-{% endblock %}
+            {% endblock %}
```
Comment (aanil) [inline dds_web/templates/user/password_reset_completed.html:39]: Why is endblock indented so much?` {% block content %}` is not indented at all.

Comment (inaod568) [reply dds_web/templates/user/password_reset_completed.html:39]: I can't even change it, cause the save just does it again

Comment (inaod568) [reply dds_web/templates/user/password_reset_completed.html:39]: Seems to be a common problem in vscode but the proposed fix is not applicable for me

File: dds_web/templates/user/password_reset_completed.html [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
-    </ul>
+  <h1>Password updated!</h1>
+  {% with messages = get_flashed_messages(with_categories=true) %} {% if
+  messages %}
+  <div class="flashes">
+    {% for category, message in messages %}
+    <div class="alert alert-{{category}}" role="alert">{{ message }}</div>
+    {% endfor %}
+  </div>
+  {% endif %} {% endwith %} {% if units_to_contact %}
```
Comment (aanil) [inline dds_web/templates/user/password_reset_completed.html:12]: Not sure if this is better 😅

**Review by aanil — APPROVED** [review]
Looks good! 👍

File: dds_web/templates/user/password_reset_completed.html (hunk 2 above)
Comment (inaod568) [reply dds_web/templates/user/password_reset_completed.html:12]: Haha, a tiny bit maybe...

**Pull request merged**